### PR TITLE
APDU10150MO sku addition

### PR DESCRIPTION
**Pull request by Anup-Jadhav**
APDU10150MO sku addition
https://cisgloballlc.atlassian.net/browse/LNX-8139

### CODE REVIEW

**Review by DineshKumar2230 — APPROVED** [review]

File: sku/CISTW2SKU.c [hunk 1]
```diff
@@ -6,39 +6,99 @@ static char buff[BUFFSIZE] = {0};
...
+	{O_T_UC, O_T_UW, O_T_LW, O_T_LC, O_T_RT, O_T_CD, O_T_CT, O_CUR13},
+	{O_T_UC, O_T_UW, O_T_LW, O_T_LC, O_T_RT, O_T_CD, O_T_CT, O_CUR16},
+	{O_T_UC, O_T_UW, O_T_LW, O_T_LC, O_T_RT, O_T_CD, O_T_CT, O_CUR13},
+	{O_T_UC, O_T_UW, O_T_LW, O_T_LC, O_T_RT, O_T_CD, O_T_CT, O_CUR16},
+	{O_T_UC, O_T_UW, O_T_LW, O_T_LC, O_T_RT, O_T_CD, O_T_CT, O_CUR13},
+	{O_T_UC, O_T_UW, O_T_LW, O_T_LC, O_T_RT, O_T_CD, O_T_CT, O_CUR16},
+	{O_T_UC, O_T_UW, O_T_LW, O_T_LC, O_T_RT, O_T_CD, O_T_CT, O_CUR13}},
+	//cb alarm
+	{ {C_C16_UC, C_C16_UW, C_C16_LW, C_C16_LC, C_C_RT, C_C_CD, C_C_CT, CB_POLE1, CB_CUR16},
+	{C_C16_UC, C_C16_UW, C_C16_LW, C_C16_LC, C_C_RT, C_C_CD, C_C_CT, CB_POLE1, CB_CUR16} }},
```
Comment (Niranjan-6) [inline sku/CISTW2SKU.c:96]: As per DA, current rating is 20Amps. Please cross check once with DA.

File: sku/CISTW2SKU.c [hunk 2]
```diff
@@ -6,39 +6,99 @@ static char buff[BUFFSIZE] = {0};
...
+	{ {1,8,2}, {2,8,2}, {3,8,2} ,{4,8,2}, {5,8,2} },
+	//unit alarm
+	{  U_T_UC, U_T_UW, U_T_LW, U_T_LC, U_T_RT, U_T_CD, U_T_CT  },
+	//phase alarm
+	{ {P_230V_UC, P_230V_UW, P_230V_LW, P_230V_LC, P_V_RT, P_V_CD, P_V_CT,
+	P_C32_UC, P_C32_UW, P_C32_LW, P_C32_LC, P_C_RT, P_C_CD, P_C_CT,
+	P_T_UC, P_T_UW, P_T_LW, P_T_LC, P_T_RT, P_T_CD, P_T_CT,
+	P_P_UC, P_P_UW, P_P_LW, P_P_LC, P_P_RT, P_P_CD, P_P_CT} },
+	//outlet alarm
+	{ {O_T_UC, O_T_UW, O_T_LW, O_T_LC, O_T_RT, O_T_CD, O_T_CT, O_CUR16},
```
Comment (Niranjan-6) [inline sku/CISTW2SKU.c:54]: Outlet 1 is C15, Outlet 2 is C19 and so on. Hence, current rating of Outlets should be changed **_for all outlets_**. First should be O_CUR13 next is O_CUR16 and so on..

**Review by Niranjan-6 — COMMENTED** [review]
Please look into the review comments and modify accordingly if required.

**Review by Niranjan-6 — APPROVED** [review]
Approved after review comments are fixed.

**Pull request merged**